packages/grid/src/vaadin-grid-sorter.js

```diff
@@ -210,11 +204,14 @@ class GridSorter extends ThemableMixin(DirMixin(PolymerElement)) {
 
     this.dispatchEvent(
       new CustomEvent('sorter-changed', {
-        detail: { shiftClick: this._shiftClick },
+        detail: { shiftClick: this._shiftClick ?? false, fromSorterClick: this._fromSorterClick ?? false },
```

Comment:
It might be safer not to use nullish coalescing, which may have Webpack version-related compatibility issues with Flow in Vaadin 23.3.

Reply:
Refactored to remove the nullish coalescing operator.